modules/graphstudio/pages/explore-graph/actions.adoc

```diff
@@ -0,0 +1,38 @@
+= Actions
+
+The Actions tab in GraphStudio is a hub for actions that affect the entire graph database.
+
+== Clear all graph data
+
+image::clear-all-graph-data.png[]
+
+This command removes all loaded data from the schema while retaining all graph schemas and queries.
+
+Your user account must have the superuser role, and you must also be in Global View to clear all data from all graphs.
+You are required to type "clear all graph data" as a confirmation phrase to avoid accidentally performing this action.
+
+This operation will take about a minute depending on the size of your graphs and your hardware capability.
+
+[CAUTION]
+====
+Clear Graph Data deletes all data from your database irreversibly. Only the schema and queries will remain.
+
+However, if you clear the graph data by accident, the data will remain in your filesystem.
+You can start over and re-load the data into the database with the "start/resume loading" button image:start-loading-button.png[].
+====
+
+== Rebuild all graph data
+
+image::rebuild-all-graph-data.png[]
+
+Rebuilding graph data is a function performed automatically by the graph engine to compact the database storage, improving performance.
```

Comment:
@frankchen189  Do we have any role restriction for rebuild functionality? If so, we should also mention it.

Reply:
Not sure if this is the definiton of rebuilding graph engine. My understanding is that it stores data in memory to disk: 

> In TigerGraph, when new data is being loaded into the graph (such as new vertices or edges), data is first stored in memory before it is saved to disk permanently. TigerGraph runs a rebuild of the Graph Processing Engine (GPE) to commit the data in memory to disk every 30 seconds, but you can also call this endpoint to trigger a rebuild immediately.